src/macaron/repo_finder/provenance_extractor.py

```diff
@@ -41,12 +50,12 @@ def extract_repo_and_commit_from_provenance(payload: InTotoPayload) -> tuple[str
             repo, commit = _extract_from_witness_provenance(payload)
 
     if not repo or not commit:
-        logger.debug(
-            "Extraction from provenance not supported for versions: predicate_type %s, in-toto %s.",
-            predicate_type,
-            payload.__class__,
+        msg = (
+            f"Extraction from provenance not supported for versions: "
+            f"predicate_type {predicate_type}, in-toto {str(type(payload))}."
         )
-        return "", ""
+        logger.error(msg)
```

Comment:
I think Macaron don't terminate on error if we failed to perform the extraction here (or in another terms, it's not considered an error to stop the analysis). Therefore, I think we should log this DEBUG instead.